orangecontrib/network/widgets/OWNxExplorer.py

```diff
@@ -423,13 +423,6 @@ def _set_combos(self):
 
             if var.is_continuous:
                 self.nodeSizeCombo.addItem(gui.attributeIconDict[gui.vartype(var)], var.name, var)
-            elif var.is_string:
-                try: value = self.graph.items()[0][var].value
-                except (IndexError, TypeError): pass
-                else:
-                    # can value be a list?
-                    if len(value.split(',')) > 1:
-                        self.nodeSizeCombo.addItem(gui.attributeIconDict[gui.vartype(var)], var.name, var)
```

Comment:
This is actually not such a bad idea, but it's just we don't use this anywhere else.